neo/Consensus/PrepareResponse.cs

```diff
@@ -1,28 +1,10 @@
-﻿using System.IO;
-
-namespace Neo.Consensus
+﻿namespace Neo.Consensus
 {
     internal class PrepareResponse : ConsensusMessage
     {
-        public byte[] Signature;
```

Comment:
We don't check the partial signature?

Reply:
We replied almost together, @erikzhang. aehuaheuaea

It is here, @shargon: https://github.com/neo-project/neo/blob/c64748ecbac3baeb8045b16af0d518398a6ced24/neo/Network/P2P/Payloads/ConsensusPayload.cs#L79